src/screens/Auth/CredentialInputScreen.js

```diff
@@ -97,34 +98,29 @@ const styles = StyleSheet.create({
     flex: 1,
     flexDirection: 'column',
     justifyContent: 'center',
+    alignItems: 'center',
     backgroundColor: colors.lightBlue,
-    paddingTop: 10,
-    paddingBottom: 10,
+    padding: 10
   },
   inputContainer: {
+    flex:1,
     backgroundColor: colors.white,
-    borderTopLeftRadius: 25,
-    borderTopRightRadius: 25,
-    borderBottomLeftRadius: 25,
-    borderBottomRightRadius: 25,
+    borderRadius: 25,
     flexDirection: 'column',
     justifyContent: 'center',
-    marginLeft: 10,
-    marginRight: 10,
     maxHeight: 530,
-    maxWidth: 369,
     paddingTop: 20,
     paddingBottom: 20,
     paddingLeft: 20,
     paddingRight: 20,
+    width: '100%'
```

Comment:
In my experience, 100% width is usually unnecessary when dealing with flex containers but I trust you

Reply:
Here's without '100%':
![screen shot 320](https://user-images.githubusercontent.com/32969087/46050311-45c0a980-c0e8-11e8-8bbd-7f8be9214fac.png)

Here's with the 100% but a maxWidth set and your height recommendation from above: 
![screen shot 321](https://user-images.githubusercontent.com/32969087/46050355-8a4c4500-c0e8-11e8-8007-cbfd24629b0b.png)

I think this is a nice balance and won't look completely ridiculously stretched out on a tablet.

Reply:
While we're here, I'm not having any issues with the keyboard covering up the inputs in android simulator. Is that still an issue for you now? Or did you just want that as cheap insurance?

Reply:
Cheap insurance I guess!

Reply:
Let's make this a new issue as well. I'm not going to have time to get to this the next few and there are other people who can step up.